librz/include/rz_cons.h

```diff
@@ -1200,6 +1217,9 @@ RZ_API int rz_line_set_hist_callback(RzLine *line, RzLineHistoryUpCb cb_up, RzLi
 RZ_API int rz_line_hist_cmd_up(RzLine *line);
 RZ_API int rz_line_hist_cmd_down(RzLine *line);
 
+RZ_API int rz_line_undo_free(void);
+RZ_API int rz_line_undo_add_entry(int offset, RZ_OWN char *deleted_text, RZ_OWN char *inserted_text, bool is_continuous);
```

Comment:
`bool` as the return, `void` for the `_free()`

Reply:
i'm not 100% sure if these should be RZ_API, but i guess these could be ok now.

Reply:
I can minimize the surface, should I do that?